includes/class-wc-calypso-bridge-addons-screen.php

```diff
@@ -0,0 +1,79 @@
+<?php
+class WC_Calypso_Bridge_Addons_Screen extends WC_Admin_Addons {
+	/**
+	 * Call API to get extensions
+	 *
+	 * @param  string $category
+	 * @param  string $term
+	 * @param  string $country
+	 *
+	 * @return array of extensions
+	 */
+	public static function get_extension_data( $category, $term, $country ) {
+		$parameters     = self::build_parameter_string( $category, $term, $country );
+		$raw_extensions = wp_remote_get(
+			'https://woocommerce.com/wp-json/wccom-extensions/1.0/search' . $parameters
+		);
+		if ( ! is_wp_error( $raw_extensions ) ) {
+			$addons = json_decode( wp_remote_retrieve_body( $raw_extensions ) )->products;
+		}
+
+		return $addons;
+	}
+
+	/**
+	 * Receives a plugin slug as plugin/plugin.php, and strips after the forward slash.
+	 * @param  string $plugin_slug full plugin slug
+	 * @return string              plugin directory name, with no attached file name
+	 */
+	public static function prepare_plugin_directory_slug( $plugin_slug ) {
+		$plugin_directory_name = explode( '/', $plugin_slug )[0];
```

Comment:
I believe this is specific to PHP 5.4. Atomic v2 is running PHP7+ so it should be OK, but I wanted to note it if you are going to be doing something similar in core.

(https://docs.woocommerce.com/document/server-requirements/ makes it sound like the PHP version is 7+ as well, but I think we are still targeting 5.2+ like WP core?)